src/main/scala/org/broadinstitute/dsde/workbench/avram/util/Logger.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.workbench.avram.util
+
+import java.util.Collections
+import com.google.cloud.MonitoredResource
+import com.google.cloud.logging.Payload.StringPayload
+import com.google.cloud.logging.{LogEntry, Logging, LoggingOptions, Severity}
+import scala.collection.JavaConverters._
+
+/*
+ Logger allows logging to Stackdriver Logging
+ */
+class Logger {
+
+  private val logging: Logging = LoggingOptions.getDefaultInstance.getService
+
+  // this could be passed to the Logger as a parameter
+  private val logName = "avram"
+
+  //these values should come from config
+  private val projectId = "broad-avram-dev"
+
+  private val resourceType = "cloud_debugger_resource"
+
+  private def entry(text: String): LogEntry = LogEntry
+    .newBuilder(StringPayload.of(text))
+    .setSeverity(Severity.INFO)
+    .setLogName(logName)
+    .setResource(MonitoredResource.newBuilder(resourceType).setLabels(Map("project_id" -> projectId, "app" -> "avram").asJava).build)
+    .build
+
+  def log(message: String) = {
+    logging.write(Collections.singleton(entry(message)))
+  }
```

Comment:
we might want the log method to take the severity also. we would also need to add that param to entry

Reply:
Google recommends using `java.util.logging.Logger` (https://cloud.google.com/appengine/docs/standard/java/logs/) which I think we should do. Adopting that interface gives us `logger.info()`, `logger.warn()`, `logger.info()`, etc.

Reply:
I attempted to use `java.util.logging.Logger` but for some reason was not able to see the log messages in Stackdriver